src/common/docker-compose/index.ts

```diff
@@ -86,6 +83,15 @@ export const generate = async (dependency_manager: LocalDependencyManager): Prom
           ...compose.services[url_safe_ref],
         };
       }
+
+      const cpu = node.node_config.getCpu();
+      const memory = node.node_config.getMemory();
+      if (cpu || memory) {
+        const service = compose.services[url_safe_ref];
+        service.deploy = { resources: { limits: {} } };
+        if (cpu) { service.deploy.resources.limits.cpus = cpu.toString(); }
```

Comment:
Do these limits apply if you don't set the `--compatability` flag for the compose
verify via `docker stats`